addons/grenades/config.cpp

```diff
@@ -10,7 +10,8 @@ class CfgPatches {
             "ACE_HandFlare_Green",
             "ACE_HandFlare_Yellow",
             "ACE_M84",
-            "ACE_M14"
+            "ACE_M14",
+            "ACE_CTS9"
```

Comment:
Unsure what the ACE guidelines are, but some projects prefer items to be prepended to lists like this to prevent unrelated lines being edited (i.e, the M14 line here).

Reply:
It's fine.